lms/djangoapps/courseware/transformers.py

```diff
@@ -0,0 +1,93 @@
+"""
```

Comment:
Why is the name of this file generic?  For example, why not call it grades_transformer.py?